flowcraft/generator/components/metagenomics.py

```diff
@@ -41,6 +41,42 @@ def __init__(self, **kwargs):
             "kraken"
         ]
 
+class Kraken2(Process):
+    """kraken2 process template interface
+
+            This process is set with:
+
+                - ``input_type``: fastq
+                - ``output_type``: txt
+                - ``ptype``: taxonomic classification
+    """
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fastq"
+        self.output_type = "txt"
+
+        self.params = {
+            "kraken2DB": {
+                "default": "'minikraken2_v1_8GB'",
+                "description": "Specifies kraken2 database."
```

Comment:
Do you need to specify a full path? relative path?

Reply:
Full path. I've clarified the param description.

Reply:
If so, why is the default not a full path? I'm assuming it's on the image? If yes, then add that information as well.